client/package.json

```diff
@@ -15,27 +15,37 @@
   },
   "scripts": {
     "start": "react-scripts start",
```

Comment:
webpack start also works :D you can remove `react-scripts` package from your project now